hindi/swami-vivekananda/teachings-of-swami-vivekananda/quotes.md

```diff
@@ -0,0 +1,10 @@
+श्री स्वामी विवेकानंद जी की शिक्षाएं|

```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.